analyzer/vulnerability-consumer/src/main/java/eu/fasten/analyzer/vulnerabilityconsumer/db/MetadataUtility.java

```diff
@@ -244,8 +244,23 @@ public Long getPackageId(String pkgName, DSLContext context) {
                 .fetch();
 
         var callIds = new HashSet<>(callInfo.map(Record2::value1));
+//        callIds.addAll(callIdsSubProj);
+        callInfo.forEach(r -> cache.callIdToPkgVsn.put(r.value1(), cache.pkgVsnIdToVsn.get(r.value2())));
+        return callIds;
+    }
```

Comment:
Modification in this file does not seems to be related to exceptions handling.
I would suggest to isolate them in a different PR if they are still relevant.